expo/gbe/models/remains.py

```diff
@@ -1931,3 +1950,7 @@ class ProfilePreferences(models.Model):
     class Meta:
         verbose_name_plural = 'profile preferences'
         app_label = "gbe"
+
+
+def mail_to_user(subject, message, user):
```

Comment:
this is a basic function call that has nothing to do with models.  Why put it in a model py file?